### DISCUSSION TITLE
Scroll with autocomplete?

### DISCUSSION THREAD

**Discussion by Huiet** [mantinedev/mantine#2919] [Q&A] [answered]
I've got an autocomplete and want to allow the user to scroll through ~30 options.

Currently I have that functionality working, by effectively limiting the height and setting overflow:
<Autocomplete
        styles={{
          dropdown: {overflowY: 'auto', maxHeight: '20em'}
        }}
        limit={30}
        ....
</Autocomplete>

The issue arises when I go to use the arrow keys to select things: once it gets to the bottom of the visible list it doesn't trigger scrolling, it just goes into the overflow portion.

I'm not sure if I can hack this together with css or if it's deeper with the component itself. Wanting to know if anyone has crossed this bridge yet!

**Comment by rtivital** [comment, marked as answer]
That is not currently supported

**Reply by Huiet** [reply]
I wonder what it would take to implement this 🤔 .
I don't have time to dive into it now, but I may circle back.
If anyone is interested in picking up the baton earlier this looked really promising at first glance:
http://jsfiddle.net/squeral/4jf0n2ff/

Would have to do something where it's implicit and likely tied to setting a specific height attribute on the element itself, but it would be a neat feature to have that could really improve ux in certain applications
